src/games/strategy/triplea/ui/MapPanel.java

```diff
@@ -95,26 +93,25 @@
   private final LinkedBlockingQueue<Tile> undrawnTiles = new LinkedBlockingQueue<>();
   private Map<Territory, List<Unit>> highlightedUnits;
   private Cursor hiddenCursor = null;
+  private MapRouteDrawer routeDrawer = new MapRouteDrawer();
 
 
   /** Creates new MapPanel */
   public MapPanel(final GameData data, final MapPanelSmallView smallView, final IUIContext uiContext,
-      final ImageScrollModel model)  {
+      final ImageScrollModel model) {
     super(uiContext.getMapData().getMapDimensions(), model);
     this.uiContext = uiContext;
     setCursor(this.uiContext.getCursor());
     this.m_scale = this.uiContext.getScale();
     this.backgroundDrawer = new BackgroundDrawer(this);
     this.tileManager = new TileManager(this.uiContext);
-    final Thread t = new Thread(this.backgroundDrawer, "Map panel background drawer");
-    t.setDaemon(true);
```

Comment:
AFAIK all threads are daemon threads by default - at least does the Javadoc say, that only deamon threads can co-exist with the JVM